### PR TITLE
[4.0] new usergroup form

### PR DESCRIPTION
**Pull request by brianteeman**
Update the markup for consistency with other similar views eg new access level form

### before
image

### after
image

### CODE REVIEW

**Comment by SharkyKZ** [comment]
That markup was added so form could be extended by plugins. See https://github.com/joomla/joomla-cms/pull/21696.

**Comment by brianteeman** [comment]
@SharkyKZ well it obviously needs to be changed in some way so at least it looks consistent. Perhaps only using the tabset if there is more than one tab?

**Comment by SharkyKZ** [comment]
IMO, just converting tabs from Bootstrap to CE would be enough.

**Comment by brianteeman** [comment]
@SharkyKZ I think you may be right. Will try to make the changes over the weekend

**Comment by brianteeman** [comment]
Updated from bs to ce as per suggestion from @SharkyKZ and updated screenshots in the original post

File: administrator/components/com_users/tmpl/group/edit.php [hunk 1]
```diff
@@ -17,15 +17,18 @@
 ?>
 
 <form action="<?php echo Route::_('index.php?option=com_users&layout=edit&id=' . (int) $this->item->id); ?>" method="post" name="adminForm" id="group-form" class="form-validate">
-	<?php echo JHtml::_('bootstrap.startTabSet', 'myTab', array('active' => 'details')); ?>
-	<?php echo JHtml::_('bootstrap.addTab', 'myTab', 'details', Text::_('COM_USERS_USERGROUP_DETAILS')); ?>
+	<?php echo HTMLHelper::_('uitab.startTabSet', 'myTab', array('active' => 'details')); ?>
+	<?php echo HTMLHelper::_('uitab.addTab', 'myTab', 'details', Text::_('COM_USERS_USERGROUP_DETAILS')); ?>		<fieldset>
```
Comment (Quy) [inline administrator/components/com_users/tmpl/group/edit.php]:
```suggestion
	<?php echo HTMLHelper::_('uitab.addTab', 'myTab', 'details', Text::_('COM_USERS_USERGROUP_DETAILS')); ?>
	<fieldset>
```

**Comment by tonypartridge** [comment]
@brianteeman can you resolve conflict and review Quy's changes then ill test

**Comment by brianteeman** [comment]
@tonypartridge conflicts resolved etc

**Comment by Quy** [comment]
@brianteeman Please check as the only change is new lines.

**Comment by brianteeman** [comment]
Closing as there were changes made elsewhere that make the original version of this redundant (and why there was nothing left  of the pr after updating the branch)

**Pull request closed without merging**